src/utils/parseAnalytics.js

```diff
@@ -51,8 +55,8 @@ export default function parseAnalytics(analytics) {
     cursePercentage: ((curseDownloadSum / downloadSum) * 100),
     twitchPercentage: ((twitchDownloadSum / downloadSum) * 100),
     pointDailyAverage,
-    estimatedRevenues: pointSum * PTS_USD_RATE,
-    dailyEstimatedRevenues: pointDailyAverage * PTS_USD_RATE,
+    estimatedRevenues: [ pointSum * PTS_USD_RATE, pointSum * PTS_GBP_RATE, pointSum * PTS_EUR_RATE, pointSum * PTS_CAD_RATE ],
```

Comment:
IMO if we want to implement different currencies the best would be to just not compute this and use the `pointSum` multiplied by the rate for each dynamically when we display it in the Stats

Reply:
I've moved the calculation over to the component rendering in commit 6b016a9.